src/content/code/plugin-api/camera/camera-viewport.ts

```
@@ -0,0 +1,32 @@
+if (viewportBounds) {
+  console.log("Viewport bounds:");
+  console.log("West:", viewportBounds.west);
+  console.log("South:", viewportBounds.south);
+  console.log("East:", viewportBounds.east);
+  console.log("North:", viewportBounds.north);
+} else {
+  console.log("Viewport is undefined.");
+}
+
+// Example 2: Check viewport and suggest adjustments
+const viewportBounds = reearth.camera.viewport;
```

Biome

[error] Shouldn't redeclare 'viewportBounds'. Consider to delete it or rename it.

'viewportBounds' is defined here:


(lint/suspicious/noRedeclare)